crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -63,6 +66,9 @@ use tower_http::cors::{AllowOrigin, CorsLayer};
 use tracing::{info, warn};
 use uuid::Uuid;
 
+/// The default maximum lag between the current timestamp and the checkpoint timestamp.
+const DEFAULT_MAX_CHECKPOINT_TS_LAG: u64 = 300_000;
```

Comment:
Can we use a `Duration` here? We've been bitten before by using u64s to represent measures of time.



```suggestion
const DEFAULT_MAX_CHECKPOINT_TS_LAG: Duration = Duration::from_secs(300);
```